Server/Plugins/InfoReg.lua

```diff
@@ -59,7 +59,7 @@ local function MultiCommandHandler(a_Split, a_Player, a_CmdString, a_CmdInfo, a_
 	-- A verb was specified, look it up in the subcommands table:
 	local Subcommand = a_CmdInfo.Subcommands[Verb]
 	if (Subcommand == nil) then
-		if (a_Level > 1) then
+		if (a_Level >= 1) then
```

Comment:
I think `a_Level + 1` makes more sense, is better to understand. Also above is used `a_Level + 1`.